File: URI/C/1006.c
#include <stdio.h>
#define WEIGHT_A 2
#define WEIGHT_B 3
#define WEIGHT_C 5

int main() {
  double A, B, C;
  double MEDIA;
  scanf("%lf %lf %lf\n", &A, &B, &C);

  MEDIA = ((A * WEIGHT_A) + (B * WEIGHT_B) + (C * WEIGHT_C)) / (WEIGHT_A + WEIGHT_B + WEIGHT_C);
  printf("MEDIA = %.1lf\n", MEDIA);
  return 0;
}

File: uri/go/1006.go
package main

import (
	"fmt"
)

func main() {

	var A float64
	var B float64
	var C float64

	fmt.Scanf("%f", &A)
	fmt.Scanf("%f", &B)
	fmt.Scanf("%f", &C)

	media := 0.2 * A + 0.3 * B + 0.5 * C
	fmt.Printf("MEDIA = %.1f\n", media)

}